BibBuddy/app/src/main/res/layout/list_view_item_note.xml

```diff
@@ -16,30 +17,88 @@
         android:orientation="horizontal">
 
         <ImageView
-                android:id="@+id/noteType"
+            android:id="@+id/note_type"
+            android:layout_width="50dp"
+            android:layout_height="50dp"
+            android:layout_centerVertical="true"
+            android:contentDescription="@string/description_text_note"
+            android:src="@drawable/document" />
+
+        <RelativeLayout
+            android:id="@+id/voice_note_layout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:visibility="gone">
+
+            <ImageButton
+                android:id="@+id/play_note"
                 android:layout_width="50dp"
                 android:layout_height="50dp"
                 android:layout_centerVertical="true"
-                android:contentDescription="@string/description_text_note"
-                android:src="@drawable/document" />
+                android:layout_marginStart="7dp"
+                android:layout_toEndOf="@+id/seekBar"
+                android:background="@null"
+                android:contentDescription="@string/play_button_note"
+                android:scaleType="fitXY"
```

Comment:
I would suggest using "fitCenter" here - "fitXY" leads to distortions on my screen.